packages/bridge/README.md

```diff
@@ -362,6 +363,39 @@ import { destroyModal } from "@getflip/bridge";
 await destroyModal({ id: "my-modal" });
 ```
 
+### Toasts
+
+#### `showToast`
+
+Shows a toast by the host app.
+
+**Param**
+
+```js
+{
+  content: string;
```

Comment:
Question: Does the wording "content" imply that HTML is allowed here? I think, otherwise we've used "text" in the past, when only plaintext is allowed.

Reply:
Right, would change it to `text` then to be less ambiguous.